.ci/azure-pipelines/env.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       Ubuntu 18.04:
         # Test the oldest supported version of Ubuntu
         UBUNTU_VERSION: 18.04
-        VTK_VERSION: 6
+        VTK_VERSION: 7
```

Comment:
This has to be done in a separate pull request, otherwise the docker images won't be updated and some checks in this PR will continue to fail

Reply:
https://github.com/PointCloudLibrary/pcl/pull/5440

Reply:
> > @mvieth , @SunBlack [ In Cmakelist under visualization folder ](https://github.com/PointCloudLibrary/pcl/blob/master/visualization/CMakeLists.txt#L231) do i need to update anything to this line? At first look, i do not think so but could you confirm
> 
> No, I would also say that this can stay as it is

Same here. The code looks like it is generally needed for VTK6 under 18.04 and has nothing to do with the legacy backend. You could test what happens if you remove it, but I think you can leave it as it is, since probably the support of VTK6 will be gone with the end of Ubuntu 18.04 next year anyway.